### PR TITLE
feat: add destroyBackendTimer to stop reporting metrics

### LINKED ISSUE #6101: [NIGNX-LUA] Add a method "destroyBackendTimer" to stop reporting metrics?
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Question or discussion
- [ ] Bug
- [ ] Requirement
- [x] Feature or performance improvement

___
### Requirement or improvement
- Please describe your requirements or improvement suggestions.

Currently we can start a backend timer to report metrics, but there is no way to stop the timer. It would be great if we can solve via adding a new method to stop timer and clean up unreported data, so that we can solve problem like https://github.com/apache/apisix/issues/2895

### PR DESCRIPTION
**Pull request by spacewander**
Close https://github.com/apache/skywalking/issues/6101

### CODE REVIEW

File: t/client.t [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+_EOC_
+
+    $block->set_value("config", $default_config);
+});
+
+run_tests;
+
+__DATA__
+
+=== TEST 1: start backend timer
```
Comment (wu-sheng) [inline t/client.t:61]: Why the following commented?

File: README.md [hunk 2]
```diff
@@ -123,6 +123,7 @@ The following APIs are for developers or using this lib out of the Nginx case.
 
 ## Nginx APIs
 - **startTimer**, `require("skywalking.client"):startBackendTimer("http://127.0.0.1:8080")`. Start the backend timer. This timer register the metadata and report traces to the backend.
+- **destroyBackendTimer**, `require("skywalking.client"):destroyBackendTimer()`. Stop the timer created by `startBackendTimer`, and clean unreported data.
```
Comment (wu-sheng) [inline README.md:126]: We need document to explain when and how to use this API.

File: t/client.t (hunk 1 above)
Comment (spacewander) [reply t/client.t:61]: What do you mean for `commented`? It is syntax for `.t` file, not the Lua.

File: README.md (hunk 2 above)
Comment (spacewander) [reply README.md:126]: @wu-sheng 
Added in the new commit.

**Review by moonming — APPROVED** [review]

**Pull request merged**